rotkehlchen/csv_exporter.py

```diff
@@ -329,7 +329,7 @@ def add_tx_gas_cost(
             timestamp=timestamp,
         )
 
-    def create_files(self, out_dir='/home/lefteris/.rotkelchen/CSV/'):
+    def create_files(self, out_dir='/home/lefteris/.rotkehlchen/CSV/'):
```

Comment:
ops that default should go away. Will handle it soon.

Reply:
Added #12 to track this one down